modules/reindex/src/main/java/org/elasticsearch/index/reindex/RestReindexAction.java

```diff
@@ -67,11 +67,12 @@ public RestChannelConsumer prepareRequest(RestRequest request, NodeClient client
             return doPrepareRequest(request, client, true, true);
```

Comment:
I think we should rename the system property (one line up, cannot comment on that one) to be "es.reindex.persistent".